src/components/Navigation/Navigation.stories.js

```diff
@@ -10,6 +10,13 @@ export default (asStory) => {
           <Navigation.Item icon="Search" label="My Active Jobs" to="/my-active-jobs" />
         </Navigation>
       ))
+      .add('with icon size', () => (
+        <Navigation color="GREEN" data-e2e-id="default-navigation">
```

Comment:
What is it for `data-e2e-id`? I don't see e2e tests.

Reply:
Fixed